packages/web/src/scss/components/Accordion/README.md

```diff
@@ -0,0 +1,86 @@
+# Accordion
+
+Accordion is a wrapper for multiple **Collapse** items, and can be open one or more at time.
+
+## Usage
+
+### Default (Always open)
```

Comment:
Please add a note about using `data-parent` as is in https://getbootstrap.com/docs/5.2/components/accordion/#always-open. There is no mention in our documentation about using it not here, nor in Collapse docs.